### ISSUE TITLE
Support for tonemapping HDR content

### ISSUE THREAD

**Issue by WillKirkby** [inalogic/pico-pixel-public#25]
At present, if I open an HDR image, pixel values above 1.0 will just be clamped.
Could we have some form of support for viewing content outside the 0.0-1.0 range?
Either via an exposure bias control to scale the values, or an optional tonemap mode (using the Reinhard operator or similar) to bring everything into a 0.0-1.0 range?

**Comment by jaytaoko** [comment]
Yes, Pico Pixel currently has no support for HDR images. However, there is a new feature on the way. It will allow you to use your own shaders in Pico Pixel to perform any kind of tone mapping you like.
I invite you to read the [blog](articles.pixelandpolygon.com) for more info on that feature.

Thank you for your feedback.
